catalog/pom.xml

```diff
@@ -43,7 +43,7 @@
             Remove restlet.version property and restlet dependencies from the dependencyManagement section when
             Solr version is updated (see DDF-1203).
         -->
-        <solr.version>5.5.0</solr.version>
+        <solr.version>6.0.0</solr.version>
```

Comment:
Looks like the restlet version and the solr saxon version are no longer used.  I removed both.